robot/Cumulus/tests/browser/zpayment_allocations/default_allocations_change_behavior.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***  
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/OpportunityPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Run keywords
+...             Disable Default And Payment Allocations
+...             Delete Records and Close Browser
+
+*** Variables ***
+&{contact1_fields}         Email=test@example.com
+&{contact2_fields}         Email=test@example.com
```

Comment:
Both the values email values are the same? Is this intentional or given by mistake?

Reply:
actually there is no use of email here just giving as some data other than just name.

Reply:
So why set it twice if the values are going to be the same. We can reuse variable1 right? Just trying to understand.

Reply:
its different contacts, names are different only email is same

Reply:
I need 2 contacts for my test since email doesn't matter, used same